newm-chain/Dockerfile

```diff
@@ -0,0 +1,23 @@
+#
+# Build
+#
+
+FROM gradle:7-jdk17 AS build
+RUN mkdir -p /home/gradle/newm-server
+COPY . /home/gradle/newm-server
+WORKDIR /home/gradle/newm-server
+RUN ./gradlew :newm-chain:shadowJar --no-daemon
+
+#
+# Run
+#
+FROM amazoncorretto:17-alpine
+
+RUN mkdir -p /opt/newm/newm-chain
+COPY --from=build /home/gradle/newm-server/newm-chain/build/libs/*.jar /opt/newm/newm-chain/newm-chain.jar
+
+WORKDIR /opt/newm/newm-chain
+USER root
```

Comment:
I believe that the parent image already uses `root` as the user default user. If that's the case, you shouldn't need to define this directive explicitly.

Ideally, one would try to avoid "root" user in containers. Here're some quick tips that you might find useful:
https://sysdig.com/blog/dockerfile-best-practices/

In particular:
- 1.1 Rootless containers
- 1.2 Don’t bind to a specific UID

Reply:
Updated PR to avoid using root - the best practices doc has some other enhancements that I'll apply as I setup publishing of the image